rpms/mysql-client.pan

```diff
@@ -1,14 +1,9 @@
 unique template rpms/mysql-client;
 
-include { 'rpms/group/core' };
-
-prefix '/software/packages';
-
-
-'{MySQL-python}' ?= nlist();
-'{mysql}' ?= nlist();
-'{mysql-connector-java}' ?= nlist();
-'{mysql-connector-odbc}' ?= nlist();
-'{perl-DBD-MySQL}' ?= nlist();
-'{perl-DBI}' ?= nlist();
-'{sqlite}' ?= nlist();
+'/software/packages' = pkg_repl('MySQL-python');
```

Comment:
For me, `pkg_repl()` was reimplemented for the YUM-based `ncm-spma` for all the reasons discussed in https://github.com/quattor/configuration-modules-core/pull/185 and the related issue https://github.com/quattor/configuration-modules-core/issues/67. For these reasons, I consider that `pkg_repl()` should be the standard way to add packages, even though using `nlist()` (or `dict()`!) works perfectly too in most of the cases.